sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/InstancesBatchResponse.Serialization.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core;
+using Azure.Iot.TimeSeriesInsights;
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
```

Comment:
Can you help me understand what the default behavior was and how we are overriding it?

Reply:
This calls for addition documentation. I will add a comment at the beginning of this class explaining why I am overriding it. Also adding similar documentation for the other classes that I am customizing.

Thanks for pointing that out.

Reply:
Okay, but you don't want this comment in a doc comment. It is meant for future  code owners to know why we made a change, not for end users (to see in tooltips and web docs).